src/sharing/components/ShareAutosuggest.jsx

```diff
@@ -126,14 +142,20 @@ export default class ShareAutocomplete extends Component {
             </div>
           )
         })}
-        <input {...props} onKeyPress={this.onKeyPress} />
+        <input {...inputProps} onKeyPress={this.onKeyPress} />
+        {loading && (
+          <Spinner
+            color={palette.dodgerBlue}
+            className={styles['recipientsSpinner']}
+          />
+        )}
```

Comment:
What is the behavior if we start typing when loading? Specially if the contact we're looking for is not loaded yet. 

Should we debounce the onKeyPress method until !loading?